lib/compare_task.rb

```diff
@@ -41,7 +41,7 @@ def render_html_comparison(description1, description2, scopes, options)
       end
     end
 
-    target = "/tmp/machinery-html-comparison"
+    target = File.join(ENV['HOME'], ".machinery", "html-comparison")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.